test/web/10-sessionService.js

```diff
@@ -0,0 +1,138 @@
+/*!
+ * Copyright (c) 2019-2020 Digital Bazaar, Inc. All rights reserved.
+ */
+
+import delay from 'delay';
+import {SessionService} from 'bedrock-web-session';
+import {login, createAccount} from './helpers.js';
+import mockData from './mock-data.js';
+
+const sessionService = new SessionService();
+
+describe('sessionService API', () => {
+  describe('unauthenticated request', () => {
+    let session;
+    beforeEach(function() {
+      session = null;
+    });
+    afterEach(async function() {
+      await sessionService.logout();
+    });
+    it('should get a session with no account', async () => {
+      let err;
+      try {
+        session = await sessionService.get();
+      } catch(e) {
+        err = e;
+      }
+      should.not.exist(err);
+      should.exist(session);
+      session.should.be.an('object');
+      const keys = Object.keys(session);
+      // an unauthenticated session has no data
+      keys.should.deep.equal([]);
```

Comment:
```suggestion
      // an unauthenticated session has no data
      session.should.eql({});
```
`eql` is a shorthand for `deep.equal` 

Please use `foo.should.have.keys(['a', 'b']);` for this sort of assertion elsewhere in this test suite.